### PR TITLE
EAS-2502 Remove unused code

### PR DESCRIPTION
**Pull request by jonathan-owens-gds**
Deprecated code used by Notify has been removed, including:

- API clients, such as 'statsd' and 'redis'
- Config, constants and code related to international billing
- Templates related to letters and branded emails
- PDF handling

---

🚨⚠️ PLEASE NOTE ⚠️🚨

After merging changes into the main branch of the utils repository, the base image will automatically be rebuilt, but then every other application image will also need to be rebuilt across environments on top of that.
This is to identify any issues that may crop up across the application as a result of making changes to utils (e.g., bumping package versions) early on.
If this is not done, it can obfuscate the origin of an issue were it to show up later.

### CODE REVIEW

**Review by gds-gharper — CHANGES_REQUESTED** [review]

File: emergency_alerts_utils/logging.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 from pythonjsonlogger.json import JsonFormatter
 
 
-def init_app(app, statsd_client=None):
+def init_app(app):
     app.config.setdefault("NOTIFY_LOG_LEVEL", "INFO")
-    app.config.setdefault("NOTIFY_APP_NAME", "none")
+    app.config.setdefault("EAS_APP_NAME", "none")
```
[the remark this answers is not in the document]
Comment (jonathan-owens-gds) [reply emergency_alerts_utils/logging.py:12]: The name of the config constant will be changed across all repos that use it (utils, admin, api) in the PRs associated with EAS-2502.

File: emergency_alerts_utils/insensitive_dict.py [hunk 2]
```diff
@@ -57,3 +57,121 @@ def make_key(original_key):
...
+
+    @property
+    def has_bad_recipient(self):
+        if self.template_type == "letter":
+            return self.has_bad_postal_address
+        return self.get(self.recipient_column_headers[0]).recipient_error
+
+    @property
+    def has_bad_postal_address(self):
+        return self.template_type == "letter" and not self.as_postal_address.valid
```
[the remark this answers is not in the document]
Comment (jonathan-owens-gds) [reply emergency_alerts_utils/insensitive_dict.py]: Deprecated methods now removed from Cell and Row classes

File: emergency_alerts_utils/insensitive_dict.py [hunk 3]
```diff
@@ -69,10 +69,10 @@ def __init__(
         *,
         index,
         error_fn,
-        recipient_column_headers,
-        placeholders,
+        # recipient_column_headers,
```
Comment (gds-gharper) [inline emergency_alerts_utils/insensitive_dict.py:72]: Is there a reason we're commenting these rather than removing them?

Comment (jonathan-owens-gds) [reply emergency_alerts_utils/insensitive_dict.py:72]: Testing - will fully remove once functionality is verified in dev

**Review by gds-gharper — APPROVED** [review]

**Pull request merged**